test/test_profile.py

```diff
@@ -89,7 +87,7 @@ def test_seed_persistance(default_params, task_manager_creator):
 
     command_sequences = []
     for _ in range(2):
-        cs = CommandSequence(url="https://example.com", reset=True)
+        cs = CommandSequence(url="https://example.com")
```

Comment:
Just realizing we're making an outbound connection here.
Could you change this to use our local server or should I do that in a different PR?

Reply:
Hm, this is also true for `test_saving` and `test_crash_profile`. I'll push a commit to fix that. Thanks for pointing it out!